internal/resources/templates/generic_results.go

```diff
@@ -24,11 +24,27 @@ const GenericResults = `
 			<div class="ui tabs divider"></div>
 		</div>
 		<div class="ui container">
+		<div class="ui grid">
+		<div class="column" style="width:65%">
 			<hr>
 			<div>
 				<pre>{{.Content}}</pre>
```

Comment:
This `pre` tag messes up the page formatting in case of long texts. For now reshuffling of content is a good enough fix, but in the long run it would be good to address this issue to avoid text overflow.

Reply:
Added `white-space: pre-wrap;` to the `pre` tag